src/modules/TTS/splitting.js

```diff
@@ -267,21 +260,9 @@
           elements.push(splitEl);
           allElements.push(splitEl);
         });
-        // insert trailing space if there was one
-        if (wholeText[wholeText.length - 1] === " ") {
-          allElements.push(createText(" "));
-        }
-        allElements.push(
-          createElement(F, "whitespace", " ", preserveWhitespace)
-        );
+        allElements.push(createText(" "));
```

Comment:
Again potentially adding space where one did not exist before.  Use previous check again:
```
// insert trailing space if there was one
if (wholeText[wholeText.length - 1] === " ") {
    allElements.push(createText(" "));
}
```

Reply:
reverted this.